modules/howtos/pages/kv-operations.adoc

```diff
@@ -71,8 +73,12 @@ include::howtos:example$kv_operations.py[tag=get_cas_replace]
 Writes in Couchbase are written to a single node, and from there the Couchbase Server will take care of sending that mutation to any configured replicas.
 The optional durability parameter, which all mutating operations accept, allows the application to wait until this replication (or persistence) is successful before proceeding.
 
-In Couchbase Server releases before 6.5, Durability was set with two options -- see the xref:2.5@python-sdk::durability.adoc[6.0 Durability documentation] -- covering how many replicas the operation must be propagated to and how many persisted copies of the modified record must exist.
-Couchbase Data Platform 6.5 refines these two options, with xref:6.5@server:learn:data/durability.adoc[Durable Writes] -- although they remain essentially the same in use.
+In Couchbase Server releases before 6.5, Durability was set with two options -- see the
+xref:2.7@python-sdk::durability.adoc[6.0 Durability documentation] --
```

Comment:
Is the xref:2.7@python-sdk implying a 2.7 version of the SDK?  Only 2.5.x exists.

Reply:
Hah, lazy assumption on my part... (we talk about 2.7 API, I assumed this should be updated to that, my bad...)
Thanks for the spot!
Reverted in  next commit.

Reply:
😆 All good!